x-pack/legacy/plugins/epm/server/ingest.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import fetch from 'node-fetch';
+import { ReturnTypeCreate, ReturnTypeUpdate } from '../../ingest/common/types/std_return_format';
+import { Datasource } from '../../ingest/server/libs/types';
+import { Request } from './types';
+
+// Temporary while we're iterating.
```

Comment:
Like the comment says, this is just a place to store Ingest-related behavior, like we do with Registry, Packages, etc. I expect these to be replaced by calls to functions from Ingest but this keeps them together while we're figuring thing out.